homeassistant/components/ecoforest/__init__.py

```diff
@@ -34,9 +35,9 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     except EcoforestAuthenticationRequired:
         _LOGGER.error("Authentication on device %s failed", host)
         return False
-    except Exception:  # pylint: disable=broad-except
+    except Exception as err:  # pylint: disable=broad-except
         _LOGGER.error("Error communicating with device %s", host)
-        return False
+        raise ConfigEntryNotReady() from err
```

Comment:
```suggestion
        raise ConfigEntryNotReady from err
```

Reply:
I thought this was required for the linting. Thanks. 👏